lib/project_types/script/messages/messages.rb

```diff
@@ -117,6 +119,9 @@ module Messages
             Usage: {{command:%s enable --API_key=<API_key> --shop_domain=<my_store.myshopify.com>}}
           HELP
 
+          info: "A script remains enabled until you manually disable it. To disable the script, "\
```

Comment:
Looking at Anette's [comment](https://github.com/Shopify/script-service/issues/1106#issuecomment-634182923),  I think we need to add `{{*}}` to the start of that string.

Reply:
I agree with both points @jacobsteves raises. That was a bit of a concern of mine, but not familiar enough with all Script options to know what's appropriate. The force option can always push and overwrite.

The tip should use the star, and didn't notice in the screen grab that it was omitted. (Slack put an > icon over that part of the snag).